src/insights_client/__init__.py

```diff
@@ -147,58 +140,55 @@ def gpg_validate(path):
     """
     # EGG= may be None or an invalid path
     if not path or not os.path.exists(path):
-        client_debug(
-            "Path '{path}' does not exist and cannot be GPG validated.".format(
-                path=path
-            )
+        logger.debug(
+            "Path '%s' does not exist and cannot be GPG validated.",
+            path
         )
         return False
 
     # EGG may be a path to a directory (which cannot be signed)
     if BYPASS_GPG:
-        client_debug(
-            "'BYPASS_GPG' is set, pretending the GPG validation of '{path}' succeeded.".format(
-                path=path
-            )
+        logger.debug(
+            "'BYPASS_GPG' is set, pretending the GPG validation of '%s' succeeded."
+            path
         )
         return True
 
     if not os.path.exists(path + ".asc"):
-        client_debug(
-            "Path '{path}' does not have an associated '.asc' file.".format(path=path)
+        logger.debug(
+            "Path '%s' does not have an associated '.asc' file.",
+            path
         )
         return False
 
     # The /var/lib/insights/ directory is used instead of /tmp/ because
     # GPG needs to have RW permissions in it, and existing SELinux rules only
     # allow access here.
+    logger.debug("Creating temporary directory in %s...", TEMPORARY_GPG_HOME_PARENT_DIRECTORY)
     home = tempfile.mkdtemp(dir=TEMPORARY_GPG_HOME_PARENT_DIRECTORY)
+    logger.debug("Created temporary directory: %s", home)
```

Comment:
I don't think we have to log every statement in the happy path, it will be too verbose. The logger statement two lines above is enough.